spec/models/web_hook_spec.rb

```diff
@@ -64,6 +64,17 @@
 
         WebHook.enqueue_hooks(:post, post_id: 1)
       end
+
+      it 'registers related event on DiscourseEvent' do
```

Comment:
I think right now it is ok. I usually try to avoid stubbing as much as possible unless it involves a call to an external API.